index.js

```diff
@@ -465,12 +466,11 @@
         if(!guild_mute_rank_staff){var guild_mute_rank_staff = ''}
         if(!guild_mute_rank_username){var guild_mute_rank_username = ''}
         client.channels.cache.get(staffChannel).send(`-----------------------------------------------------\n**${guild_mute_rank_staff} ${guild_mute_staff}** has muted **${guild_mute_rank_username} ${guild_mute_username}** for **${guild_mute_time}**\n-----------------------------------------------------`)
-        // let serverID = "522586672148381726";
-        // let displayNickname = `${guild_mute_username}`;
-        // let serverMembers = client.guilds.cache.get(serverID).members
-        // let matchedMember = serverMembers.cache.find(m => m.displayName === displayNickname);
-        // if (!matchedMember) {return}
-        // matchedMember.roles.add('529453283782164502')
+        let displayNickname = guild_mute_username;
+        let serverMembers = client.guilds.cache.get(serverID).members.cache;
+        let matchedMember = serverMembers.findKey(user => user.nickname == displayNickname);
+        if (!matchedMember) {return}
+        serverMembers.get(matchedMember).roles.add('529453283782164502');
```

Comment:
Please make sure this function will work as I am pretty sure you need to get a role from cache

Reply:
it does - just tested it